arxiv/base/static/sass/_custom.sass

```diff
@@ -212,3 +212,21 @@ tr.is-warning
 
 .pagination-list li + li // fixes override from content li + li
   margin: 0
+
+/* flash notifications */
+
+.notifications
+  margin-top: 5px
+
+.notification
+  padding: 0.25em 0.5em 0.25em 0.5em
```

Comment:
notification padding is a Bulma variable that can be set in arxivstyle.sass - use `$notification-padding`